test/contracts/workers/set_is_worker.js

```diff
@@ -94,11 +94,11 @@ module.exports.perform = (accounts) => {
   it('pass to set worker if deactivation height is equal to current block number + 1', async () => {
 
     let blockNumber = await web3.eth.getBlockNumber();
-    deactivationHeight = blockNumber + 1;
+    deactivationHeight = blockNumber + 2; //It is incremented by 2 becoz when u make an smart contract method call using web3's call,it increments by 1 as it simulates what sendTransaction is performing.
```

Comment:
I think this comment may be misleading. The reason why we increment by 2 is because of both: (a) block.number returns the pending block number (which is 1 more than the latest block number, which is what is returned by getBlockNumber) and (b) the conditions of the setWorker function that we are testing here.

Please reword.